src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {string, node, oneOf} from 'prop-types';
+import style from './Thumbnail.st.css';
+import Check from 'wix-ui-icons-common/Check';
+import {Thumbnail as CoreThumbnail, ThumbnailProps as CoreThumbnailProps} from 'wix-ui-core/Thumbnail';
+import {withStylable} from 'wix-ui-core/withStylable';
+import {Text} from '../Text';
+
+export interface ThumbnailProps extends CoreThumbnailProps {
+  /** image of the thumbnail */
+  image?: React.ReactElement<any>;
+  /** title of the thumbnail */
+  title: string;
+  /** description of the thumbnail */
+  description?: string;
+}
+
+const StyledThumbnail = withStylable<CoreThumbnailProps>(
+  CoreThumbnail,
+  style,
+  () => ({})
+);
+
+export const Thumbnail: React.SFC<ThumbnailProps> = props => {
+  const {children, title, image, description, ...rest} = props;
+
+  return (
+      <StyledThumbnail {...rest} selectedIcon={<Check size="24"/>}>
+        <div className={style.container}>
```

Comment:
Why do you need another wrapping container?
You can add styling to the core's `.root`.

Reply:
The `children` prop according to core Thumbnail must be one element (Children.only()), so it must be wrapped.